python/galois/CMakeLists.txt

```diff
@@ -16,8 +16,13 @@ add_library(_pagerank MODULE ${PAGERANK_SOURCES})
 python_extension_module(_pagerank)
 target_link_libraries(_pagerank Galois::shmem)
 
+add_cython_target(_connectedComponents _connectedComponents.pyx shmem.pyx CXX OUTPUT_VAR CONNECTEDCOMPONENTS_SOURCES)
```

Comment:
I think once we enter python land, components should have snake_case names (if we have a choice) to match the convention in that ecosystem.

Reply:
This.

Reply:
Just to elaborate on this point, in Python, an underscore prefix is commonly used to denote internal APIs that library developers don't intend end-users to ever see or use.